justfile

```diff
@@ -2,6 +2,8 @@
 # ^ A shebang isn't required, but allows a justfile to be executed
 #   like a script, with `./justfile test`, for example.
 
+set windows-powershell := true
```

Comment:
i dunno what this does, but it'd heavily irritate me when i see this.

Reply:
See https://github.com/casey/just#windows-powershell

By default, just runs sh on Windows, which means you must build just inside a cygwin or [WSL](https://docs.microsoft.com/en-us/windows/wsl/) prompt. 

This defaults the file to use powershell instead, which works from my limited testing.

Reply:
i won't be less confused when you tell me things about Windows. ;-)

Reply:
> i dunno what this does, but it'd heavily irritate me when i see this.

This isn't a super helpful comment. There are a lot of Windows users out there!